addons/test_mail_full/tests/test_portal.py

```diff
@@ -145,37 +145,6 @@ def test_portal_avatar_with_hash_pid(self):
         self.assertEqual(placeholder_response.headers.get('Content-Type'), 'image/png')
         self.assertRegex(placeholder_response.headers.get('Content-Disposition', ''), r'placeholder\.png')
 
-    def test_portal_message_fetch(self):
-        """Test retrieving chatter messages through the portal controller"""
-        self.authenticate(None, None)
-        message_fetch_url = '/mail/chatter_fetch'
```

Comment:
Why remove test?

Reply:
The purpose of this test is to check that the returned message count (which was displayed at the top of the old chatter widget) is correct. We don't have it in the backend chatter design.